vassal-app/src/main/java/VASSAL/build/module/PlayerRoster.java

```diff
@@ -496,6 +496,8 @@ public void finish() {
       if (GameModule.getGameModule().isMultiplayerConnected()) {
         final Command c = new Chatter.DisplayText(GameModule.getGameModule().getChatter(), Resources.getString(GlobalOptions.getInstance().chatterHTMLSupport() ? "PlayerRoster.joined_side_2" : "PlayerRoster.joined_side", GameModule.getGameModule().getPrefs().getValue(GameModule.REAL_NAME), translateSide(newSide)));
         c.execute();
+        GameModule.getGameModule().getServer().sendToOthers(c);
+
```

Comment:
Mark, the preferred way to do this is:

GameModule.getGameModule().sendAndLLog(c);

Which handles writing the Command into a log if it is currently open, and handles pausing of logging. You should generally never call getServer().sendToOthers() directly.

Reply:
Hi @riverwanderer ,

I would say that these sort of messages should go into the log if they are being sent to other players, so sendAndLog() is the preferred way to go for the existing messages as well.

Reply:
Is there a distinction that needs to be made between the messaging and the actual side change commands? Eg at the moment change of side is appended with messaging. I suspect that sendToOthers has been used to prevent undo.

Reply:
Changes made and given a basic test (2 different users exchanging a log file). Messages appear when expecrted.